src/plugins/data_view_management/public/components/edit_index_pattern/edit_index_pattern.tsx

```diff
@@ -91,11 +90,34 @@ export const EditIndexPattern = withRouter(
       },
     });
 
-    const timeFilterHeader = i18n.translate(
-      'indexPatternManagement.editIndexPattern.timeFilterHeader',
+    const isRollup = new URLSearchParams(useLocation().search).get('type') === 'rollup';
+    const displayIndexPatternEditor = showEditDialog ? (
+      <IndexPatternEditor
+        onSave={(iPattern) => {
+          history.push(`patterns/${iPattern.id}`);
```

Comment:
It seems this is not needed during edit since we are already on that page and this only breaks browser history.

Or am I missing something?

Reply:
fixed.